docs/HOWTO-use-dynamicassignment-batches.md

```diff
@@ -0,0 +1,43 @@
+# Dynamic Assignment Batches Framework
+
+Every Spoke campaign needs a method for assigning available texts to texters. This can be done with manual or dynamic assignment. 
+
+Dynamic assignment allows admin to use a link that once clicked gives texters access to text in the associated campaign. Texters are then able to request new texts using a request button in the dynamic assignment sidebox.
+
+## Settings to enable and enhance dynamic assignment
+- **The Dynamic Assignment Controls setting in the Texter UI Defaults panel in settings:** Needed to expose the Texter interface for using dynamic assign. This setting also allows you to choose what the text of the "request texts" button will look like.
+- **The Release Contacts setting in the Texter UI Defaults panel in settings:** Allows texters to click a second button to release their texts which unassigns them from the initial texts they have left. You can also toggle a setting to allow texters to release conversations as well as initial texts. This pairs very nicely with dynamic assign, though can also be used for manual assignment.
+- **`MAX_TEXTERS_PER_CAMPAIGN` Environment Variable:** Limit the number of texters allowed in a given campaign.
+
+## Dynamic assignment strategies
+
+By default, dynamic assignment uses the `vetted-texter` strategy for deciding which texters can request batches. This allows all **vetted** texters to request and receive batches. Strategies options can be found or added in `src/integrations/dynamicassignment-batches`. You can configure which strategies are active with `env var??`
```

Comment:
default is `finished-replies`

env var is `DYNAMICASSIGNMENT_BATCHES` (e.g. DYNAMICASSIGNMENT_BATCHES=finished-replies)